app/src/components/results/ResultList.js

```diff
@@ -53,57 +55,100 @@ class ResultList extends Component {
 
   render() {
     const { classes, query, onRowClick, selectedIdx } = this.props;
+    let tableHeader = (
+      <TableRow>
+        <CustomTableCell>Type</CustomTableCell>
+        <CustomTableCell>Name</CustomTableCell>
+        <CustomTableCell>Namespace</CustomTableCell>
+        <CustomTableCell>Created At</CustomTableCell>
+      </TableRow>
+    );
+
+    if (query.isQSL) {
```

Comment:
this render fn is getting pretty long, might it make sense to break it down into a few helper fns or addl component methods? maybe:
* generateQslTable
* generateKeywordTable

or some other breakdown that might make more sense

Reply:
sure